app/ducks/hip2Reducer.js

```diff
@@ -0,0 +1,26 @@
+export const SET_HIP2_PORT = 'app/hip2/setHip2Port';
+
+function reduceSetPort(state, action) {
+  const { payload: port } = action;
+
+  return {
+    ...state,
+    port 
+  };
+}
+
+
+function getInitialState() {
+  return {
+    port: 9892
+  }
+}
+
+export default function hip2Reducer(state = getInitialState(), action) {
```

Comment:
Packaged builds no longer start (try it out with `npm run package-linux` or whichever os you're running).

`ducks/nodeReducer.js` needs to be added to `scripts/package.sh` as well as `scripts/package.js`.